modules/subdyn/src/SubDyn.f90

```diff
@@ -1085,6 +1085,11 @@ SUBROUTINE SD_Input(SDInputFile, Init, p, ErrStat,ErrMsg)
 ! Reading reaction lines one by one, allowing for 1, 7 or 8 columns, with col8 being a string for the SSIfile
 do I = 1, p%nNodes_C
    READ(UnIn, FMT='(A)', IOSTAT=ErrStat2) Line; ErrMsg2='Error reading reaction line'; if (Failed()) return
+   j = index(line, achar(13))
```

Comment:
Can you add a note here about the Flang compiler issues so we don't remove this on accident later?

Reply:
Added comment in latest commit